src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,6 +9,125 @@ i18nReady: false
 
 [CloudCannon](https://cloudcannon.com) is a Git-based content management system that provides a visual editor for your content.
 
+## Introduction
+
+This guide will walk through the steps required to get your Astro site built, editable and live on CloudCannon. CloudCannon makes it easy to store your content in your Git repository and have non-developers update it with a live visual editing interface.
+
+## Prerequisites
+
+To get started you will need to have the following:
+
+1. **An Astro project** - If you don’t have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+## Sync your files
+
+The first step for connecting any site is to get your file system linked to CloudCannon. CloudCannon will need to clone your files to disk before anything else — this is the same process as if you were working locally in your development environment.
+
+CloudCannon offers multiple ways of connecting your files:
+
+- Git syncing through [GitHub](https://cloudcannon.com/documentation/articles/connecting-a-github-repository-as-your-source/), [Bitbucket](https://cloudcannon.com/documentation/articles/connecting-a-bitbucket-respository-as-your-source/), [GitLab](https://cloudcannon.com/documentation/articles/connecting-a-gitlab-respository-as-your-source/) and [self-hosted GitLab](https://cloudcannon.com/documentation/articles/connecting-a-self-hosted-gitlab-repository-as-your-source/).
+- [Uploading a folder or zip of your files](https://cloudcannon.com/documentation/articles/add-files-with-direct-upload/).
+
+It is highly recommended that you use the Git syncing option. This allows your updates on CloudCannon to push back to your repository. Your editors can then contribute to the Software Development Life Cycle (SDLC). This step is configured through an OAuth connection with the relevant partner. 
+
+Check out CloudCannon’s [connecting your first site](https://cloudcannon.com/documentation/articles/connecting-your-first-site/) guide for more information on this step.
+
+## Build your site
```

Comment:
I am very unclear what is happening here. Is this in a CloudCannon dashboard or on CloudCannon... somewhere?

Again, ideally this guide will only cover Astro-specific topics, and this all sounds like getting *any* site built and previewed in CloudCannon. I think we *maybe* even could get away with again front-loading pre-requisites and saying that you need an existing Astro site connected Cloud Cannon, successfully built and previewed? 

I know that sounds a little extreme, and I'm not entirely sold on it. But... I'm thinking about it.  :sweat_smile: